docroot/modules/humsci/hs_field_helpers/src/Plugin/media/Source/AudioEmbedField.php

```diff
@@ -0,0 +1,259 @@
+<?php
+
+namespace Drupal\hs_field_helpers\Plugin\media\Source;
+
+use Drupal\audio_embed_field\ProviderPluginInterface;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Entity\EntityFieldManagerInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Field\FieldTypePluginManagerInterface;
+use Drupal\media\MediaInterface;
+use Drupal\media\MediaSourceBase;
+use Drupal\media\MediaTypeInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides media source plugin for audio embed field.
+ *
+ * This file should be removed when
+ * https://www.drupal.org/project/audio_embed_field/issues/2955601 is resolved.
+ *
+ * @MediaSource(
+ *   id = "audio_embed_field",
+ *   label = @Translation("Audio embed field"),
+ *   description = @Translation("Enables audio_embed_field integration with
+ *   media."), allowed_field_types = {"audio_embed_field"},
+ *   default_thumbnail_filename = "audio.png"
+ * )
+ */
+class AudioEmbedField extends MediaSourceBase {
```

Comment:
Can this not go into stanford_media at some point?

Reply:
We removed it from Stanford media because oembed will be capable of handling sound cloud in the future